src/tests/userDashboard/actions/index.test.js

```diff
@@ -152,4 +155,46 @@ describe('Test suite for user dashboard actions', () => {
       expect(store.getActions()).toEqual(expectedActions);
     });
   });
+  it('should handle USER_IS_MEMBER', () => {
```

Comment:
If this is an action then, it should `dispatch` seems more direct. Same goes for the others below.

Reply:
Thanks for the feedback. I have renamed the description.